Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG BUILD_ENV=development
 
 # Env setup:
 ENV PYTHONPATH /app
-ENV DATABASE_URL postgres://metashare@postgres:5432/metashare
+ENV DATABASE_URL postgres://metecho@postgres:5432/metecho
```

Comment:
@wlonk I think this change broke my DB -- something about the "metecho" role not existing. I just blew up my DB and re-created, but is there a way to make this change to an existing DB?

Reply:
Three steps:

1. Connect to DB.
2. `ALTER ROLE metashare RENAME TO metecho;`
3. Change `DATABASE_URL` as you did in this commit.

Reply:
"Connect to DB" has additional prepreqs. Because we build our docker image without `psql`, we need to add that. So:

1. `./derrick shell`
2. `apt-get update && apt-get install postgresql -y`
3. `python manage.py dbshell`

Reply:
… and you can't rename the user you're connected as. Lemme rethink this.

Reply:
OK, new plan:

```
./derrick shell
> apt-get update && apt-get install postgresql -y
> psql -h postgres
>> CREATE ROLE metecho WITH SUPERUSER CREATEROLE CREATEDB REPLICATION BYPASSRLS LOGIN;
>> ^D
> psql -h postgres -U metecho
>> create database temp;
>> \c temp;
>> ALTER DATABASE metashare RENAME TO metecho;
>> \c metecho;
>> drop database temp;
>> ^D
> ^D
# done!
```

Then change values in `.env`.

Reply:
(To clarify, the `>` and `>>` indicate that you're in increasingly nested shells, and are not part of the commands to issue.)

Reply:
Sounds good -- though I don't think any values in `.env` need changing?